Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
+        return number_1 % number_2
```

Comment:
Can we add a check for negative numbers and a decision for what modulo to return? For instance, -10 % 7 is either -3 or 4. Print statement is fine as is.

Reply:
When I ran the code the % does not show up as one of the options in the message asking the user to choose an operation,  This should be added to that message.

Reply:
Adding a modulo to the list of operator options is a good idea but also remembering to show it to the user in the message.

Reply:
I agree. since mods can return either, there should be a way to determine that